docs/source/gbd2019_models/risk_attributable_causes/protein_energy_malnutrition/index.rst

```diff
@@ -4,6 +4,72 @@
 Protein Energy Malnutrition
 ===========================
 
+Protein-energy malnutrition (PEM) is an acute form of undernutrition. 
+Undernutrition can take a variety of forms, including stunting (low height for 
+age), wasting (low weight for height), underweight (low weight for age), and 
+micronutrient deficiencies (a lack of key vitamins or minerals). PEM refers 
+specifically to an acute (as opposed to chronic) disorder that results in 
+wasting, and is classified as either marasmus, kwashiorkor, or some 
+combination of these. [UpToDate-malnutrition]_ [WHO-Malnutrition]_ 
+[emedicine-PEM]_
+
+Marasmus is characterized by a low weight-for-height and reduced mid-upper arm 
```

Comment:
Here you can add more information about Marasmus and cite Dispasquale _Nutrients_ 2020:

```suggestion
The term “marasmus” is inferred from the Greek word “marasmus”, correlating to wasting or Marasmus is characterized by a low weight-for-height and reduced mid-upper arm withering. Marasmus is the most frequent syndrome of acute malnutrition [4]. It is due to inadequate energy intake over a period of months to years. It results from the body’s physiologic adaptive response to starvation in response to severe deprivation of energy and all nutrients, and is characterized by wasting of body tissues, particularly muscles and subcutaneous fat, and is usually a result of severe restrictions in energy intake. Children younger than five years are the most commonly involved because of their increased caloric requirements and increased susceptibility to infections. [reference Dispasquale _Nutrients_ 2020]
```